core/src/main/scala/flow/Generator.scala

```diff
@@ -66,6 +66,9 @@ class Generator[+T, +Out](val source: () ⇒ Future[Source[T, Future[Out]]]) {
   def recoverWithMaterializedValue[Out2 >: Out](p: PartialFunction[Throwable, Future[Out2]])(implicit ec: ExecutionContext): Generator[T, Out2] =
     use(() ⇒ source().map(_.mapMaterializedValue(outFuture ⇒ outFuture.recoverWith[Out2](p))))
 
+  def via[U](flow: Graph[FlowShape[T, U], akka.NotUsed])(implicit ec: ExecutionContext): Generator[U, Out] =
```

Comment:
It would be nice to add scaladoc here so that somebody unfamiliar with the flow library might better understand why they would use this function.

Reply:
datto-flow isn't really meant to abstract away akka-stream, only to provide some useful helper functions on top of it.

Arguably everything should have scaladoc, I guess, but this doesn't seem more deserving than anything else. The types seem pretty self-descriptive.